utils/pipeline/pipeline.go

```diff
@@ -62,14 +62,11 @@ func FanIn[T any](ctx context.Context, channels ...<-chan T) <-chan T {
 	return out
 }
 
-func Bridge[T any](ctx context.Context, chanCh <-chan <-chan T) <-chan T {
-	out := make(chan T)
+func Bridge[T any](ctx context.Context, out chan T, chanCh <-chan <-chan T) {
 	if chanCh == nil {
-		close(out)
-		return out
+		return
 	}
 	go func() {
```

Comment:
I think we should remove `go` here and let the caller decide how to run it. In this way, the caller can know when this function stop in order to close the channel properly and avoid panic.